src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUnitTest.java

```diff
@@ -1,13 +1,225 @@
 package org.broadinstitute.hellbender.tools.funcotator;
 
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
 import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
 
 /**
  * Unit tests for the {@link org.broadinstitute.hellbender.tools.funcotator}.
  * Created by jonn on 8/22/17.
  */
 public class FuncotatorUnitTest extends GATKBaseTest {
 
+    /**
+     * Data provider for the unit test for checkIfAlreadyAnnotated. In this case, we have a
+     * "Funcotator" column in the vcf header, which indicates
+     * that this vcf has already been annotated and thus the program
+     * should run on this input.
+     *
+     */
+    @DataProvider
+    Object[][] provideDataForAnnotationCheckWrong() {
+        final Set<VCFHeaderLine> headerLines = new LinkedHashSet<>();
+
+        headerLines.add(new VCFHeaderLine("fileformat", "fileformat"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
```

Comment:
You probably don't need quite so many non-Funcotator header lines here -- 4-5 would be sufficient :)